Documentation/Observers.md

```diff
@@ -169,128 +169,84 @@ All settings are optional, ***except target OR targetType***, and can be omitted
+    <!-- Time window in which max dumps per process, per observed metric can occur. See AppObserverMaxProcessDumps. -->
+    <Parameter Name="AppObserverMaxDumpsTimeWindow" DefaultValue="04:00:00" />
+    <!-- Concurrency/Parallelism Support.
+         Note: This will only add real value (substantial) on machines with capable hardware (>= 8 logical processors). FO will not attempt to do anything concurrently if
+         the number of logical processors is less than 4, even if this is set to true. If you do run on capable hardware and want AppObserver to complete a run much faster than at sequential speed, then keep this enabled.
+         You can control the level of concurrency by setting the AppObserverMaxConcurrentTasks setting. -->
+    <Parameter Name="AppObserverEnableConcurrentMonitoring" DefaultValue="true" />
+    <!-- The default value is 1/4 of detected logical processors (LPs). This would mean given 20 LPs, the number of threads that will be created will be at least 5 (and seldom more than that), if possible.
+         You can set this to -1 (unlimited), or some integer value that makes sense based on your CPU configuration, how many services AppObserver is monitoring, how comfortable you are with FO process
+         eating some additional CPU and Memory to complete the parallelized monitoring on a node with lots of services (>= 100). The impact of parallelization on nodes with less than 100 services is minimal.
+         Please test and choose a value that suits your needs or simply leave this unset and go with the default. Please see Observers.md for more information. -->
+    <Parameter Name="AppObserverMaxConcurrentTasks" DefaultValue="" />
+    <!-- KVS LVID Usage Monitoring - Windows-only.
+         NOTE: If you monitor stateful Actor services, then you should set this to true.
+         This is a temporary monitor since SF will eventually employ an updated Windows ESE libray that supports long.MaxValue number of LVIDs. -->
+    <Parameter Name="AppObserverEnableKvsLvidMonitoring" DefaultValue="false" />
+    <!-- Process Working Set type for AppObserver to monitor. 
+         Setting this to false will enable Full Working Set monitoring (Private + Shared process memory). 
+         NOTE: If you have *several* service processes of the **same name**, setting this to true will add *significant* time-to-completion and increased CPU usage (small, but more than necessary) to AppObserver. 
+         FO will automatically compute Shared + Private memory usage if it detects 50 or more same-named service processes, regardless of this setting.-->
+    <Parameter Name="AppObserverMonitorPrivateWorkingSet" DefaultValue="true" />
+    <!-- AppObserver can monitor services with resource governance limits set and put related services into Warning if they reach 90% of the specified limit (only Memory limit is supported in this release). -->
+    <Parameter Name="AppObserverMonitorResourceGovernanceLimits" DefaultValue="true" />
```

Comment:
I believe this should be "true" by default, but just wanted to confirm that it is intentionally enabled?